charts/spring-boot/spring-boot-chaos/experiment.yaml

```diff
@@ -0,0 +1,166 @@
+      # Create and monitor the experiment & helper pods
+      - apiGroups: [ "" ]
+        resources: [ "pods" ]
+        verbs: [ "create","delete","get","list","patch","update", "deletecollection" ]
+      # Performs CRUD operations on the events inside chaosengine and chaosresult
+      - apiGroups: [ "" ]
+        resources: [ "events" ]
+        verbs: [ "create","get","list","patch","update" ]
+      # Fetch configmaps details and mount it to the experiment pod (if specified)
+      - apiGroups: [ "" ]
+        resources: [ "configmaps" ]
+        verbs: [ "get","list", ]
+      # Track and get the runner, experiment, and helper pods log
+      - apiGroups: [ "" ]
+        resources: [ "pods/log" ]
+        verbs: [ "get","list","watch" ]
+      # for creating and managing to execute comands inside target container
+      - apiGroups: [ "" ]
+        resources: [ "pods/exec" ]
+        verbs: [ "get","list","create" ]
+      # deriving the parent/owner details of the pod(if parent is anyof {deployment, statefulset, daemonsets})
+      - apiGroups: [ "apps" ]
+        resources: [ "deployments","statefulsets","replicasets", "daemonsets" ]
+        verbs: [ "list","get" ]
+      # deriving the parent/owner details of the pod(if parent is deploymentConfig)
+      - apiGroups: [ "apps.openshift.io" ]
+        resources: [ "deploymentconfigs" ]
+        verbs: [ "list","get" ]
+      # deriving the parent/owner details of the pod(if parent is deploymentConfig)
+      - apiGroups: [ "" ]
+        resources: [ "replicationcontrollers" ]
+        verbs: [ "get","list" ]
+      # deriving the parent/owner details of the pod(if parent is argo-rollouts)
+      - apiGroups: [ "argoproj.io" ]
+        resources: [ "rollouts" ]
+        verbs: [ "list","get" ]
+      # for configuring and monitor the experiment job by the chaos-runner pod
+      - apiGroups: [ "batch" ]
+        resources: [ "jobs" ]
+        verbs: [ "create","list","get","delete","deletecollection" ]
+      # for creation, status polling and deletion of litmus chaos resources used within a chaos workflow
+      - apiGroups: [ "litmuschaos.io" ]
+        resources: [ "chaosengines","chaosexperiments","chaosresults" ]
+        verbs: [ "create","list","get","patch","update","delete" ]
+    image: "litmuschaos/go-runner:latest"
+    imagePullPolicy: Always
+    args:
+      - -c
+      - ./experiments -name spring-boot-chaos
+    command:
+      - /bin/bash
+    env:
+      - name: TOTAL_CHAOS_DURATION
+        value: '15'
+
+      ## Period to wait before injection of chaos in sec
+      - name: RAMP_TIME
+        value: ''
+
+      ## env var that describes the library used to execute the chaos
+      ## default: litmus. Supported values: litmus
+      - name: LIB
+        value: 'litmus'
+
+      # Chaos Monkey Spring Boot configuration
+      # Level determines at which frequency the assault happens on the application. For a value N, the assaults happens every N requests
```

Comment:
```suggestion
      # Level determines at which frequency the assault happens on the application. For a value N, the assaults happen for every N requests
```

Reply:
I don't agree with this change, the initial formulation is correct for me.